Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Docker image for Drone's terraform deployment plugin
+#
+#     CGO_ENABLED=0 go build -a -tags netgo
+#     docker build --rm=true -t plugins/drone-terraform .
+
+FROM gliderlabs/alpine:3.2
+RUN apk-install ca-certificates
+
+ENV TERRAFORM_VERSION 0.6.9
+
+RUN apk update && \
+    wget -q "https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-2.21-r2.apk" && \
+    apk add --allow-untrusted glibc-2.21-r2.apk && \
```

Comment:
if the terraform binary is statically compiled there should be no need to put glibc (or compile with musl) on the host machine, since a static Go binary will have everything it needs (except an nsswitch.conf) but the rest of the OS will get ignored.

what about compiling terraform as part of the drone build process with `--ldflags '-extldflags "-static"` so to avoid the glibc dependency, and then adding to the Docker image directly?